### ISSUE TITLE
changing arm-unknown-linux-gnueabi to hard float

### ISSUE THREAD

**Issue by justwantin** [crosstool-ng/crosstool-ng#724]
I am using slackware-14.2 and crosstool-NG version crosstool-ng-1.22.0

How would I configure arm-unknown-linux-gnueabi to hard float? I believe I would need to add the line "-mfloat-abi=hard into the configuration but I have not been able to find where to do this in menuconfig

**Comment by KirillSmirnov** [comment]
In menuconfig proceed to "Target options". Next, in "Floating point" menu specify the exact type of hard float you wish to use: hardware or softfp (despite the name, softfp really emits FPU opcodes but keeps softfp ABI).

**Comment by stilor** [comment]
There is an option, "Floating point", that does exactly that.

**Comment by justwantin** [comment]
Thanks to both you guys for responding. I found that target option and set up FPU pluse added -mfloat-abi=hard However my build failed with  gdb problem and I haven't had time to goe back and have a look at why. In any case I built a toolchain using the armv7-rpi2-linux-gnueabihf sample which got me going compiling for a banana pro so all good. Id like to figure out why my arm-unknown-linux-gnueabihf builfd didn't work but will have to come back to it on a rainy day.
